tests/sentry/monitors/endpoints/test_organization_monitor_stats.py

```diff
@@ -8,7 +8,7 @@
 
 
 @region_silo_test(stable=True)
-@freeze_time()
+@freeze_time("2023-02-08 07:57:00")
```

Comment:
I don't recommend pinning to an absolute time. It can break in unexpected ways due to things like retention